src/Uuid/Uuids.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Kirby\Uuid;
+
+use Kirby\Cache\Cache;
+use Kirby\Cms\App;
+
+/**
+ * @package   Kirby Uuid
+ * @author    Nico Hoffmann <nico@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ */
+class Uuids
+{
+	/**
+	 * Get instance for lookup cache
+	 */
+	public static function cache(): Cache
+	{
+		return App::instance()->cache('uuid');
+	}
+
+	/**
+	 * Populates cache with Uuids for all identifiable models
+	 * that need to be cached (not site and users)
+	 */
+	public static function populate(string $type = 'all'): void
```

Comment:
I think it would be great to name the possible values for the `$type` property like the UUID schemes (so `all`, `page`, `file`, `block`, `struct`). One reason is consistency, but it also makes it easier to connect different custom logic without having to employ a lookup table between scheme and `$type` value.